src/MountedTraversal.js

```diff
@@ -49,9 +49,8 @@ export function instHasClassName(inst, className) {
   if (!isDOMComponent(inst)) {
     return false;
   }
-  let classes = findDOMNode(inst).className || '';
-  classes = classes.replace(/\s/g, ' ');
-  return ` ${classes} `.indexOf(` ${className} `) > -1;
+  let classes = findDOMNode(inst).classList;
```

Comment:
Ah, yep good find, didn't think to check the browser support. Given that, and also in the interest of merging prior to #421 since this is still something that currently breaks testing, would just adding a check that classes is of type String, and returning false if it isn't, be enough?